src/annotator/integrations/html.js

```diff
@@ -105,40 +108,44 @@ export class HTMLIntegration {
     const padding = 10;
     const rightMargin = sidebarWidth + padding;
 
-    // nb. Adjusting the body size this way relies on the page not setting a
```

Comment:
The change here is to wrap the bulk of this function in a `preserveScrollPosition(...)` call.